### PR TITLE
for check

### PR DESCRIPTION
**Pull request by AndriiMan**
for check

### CODE REVIEW

**Comment by AndriiMan** [comment]
Found, but have new problem.
When i use this DateTimeFormatter class, and print minute, i have only one number, when minutes are from 01 to 09.
For example: when i have 6:08 on my computer, in my app i have 6:8.
So is my program is correct or not?

File: src/com/company/Main.java [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
-        System.out.format("%tB %te, %tY%n", calendar, calendar, calendar);
-        System.out.format("%tl:%tM %tp%n", calendar, calendar, calendar);
+        LocalDateTime date = LocalDateTime.now();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMMM d" + ", " + "Y" + "\n"
+                + "h" + ":" + "m ");
+        DateTimeFormatter formatterForAmPm = DateTimeFormatter.ofPattern("a");
+        String dateTime = date.format(formatter);
+        String dateTimeAm = date.format(formatterForAmPm);
+        String dateTimeRes = dateTime.concat(dateTimeAm.toLowerCase());
+        System.out.println(dateTimeRes);
```
Comment (AnnaDremluga) [inline src/com/company/Main.java:24]: Please, add task and the result of the program to readme.

I don't see  'Current Date and Time :'

File: src/com/company/Main.java [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
 
+    //for check
     public static void main(String[] args) {
-        Calendar calendar = Calendar.getInstance();
-        System.out.println("Current Date and Time :");
-        System.out.format("%tB %te, %tY%n", calendar, calendar, calendar);
-        System.out.format("%tl:%tM %tp%n", calendar, calendar, calendar);
+        LocalDateTime date = LocalDateTime.now();
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMMM d" + ", " + "Y" + "\n"
+                + "h" + ":" + "m ");
```
Comment (AnnaDremluga) [inline src/com/company/Main.java:19]: Why do you creating this pattern, which can be a single string (or strings for 2 patterns)?

What do these letters in pattern mean? Why some of them are big and some - small? Did you create a pattern correctly? Answer these question and you'll see how to fix the bug.

File: src/com/company/Main.java [hunk 3]
```diff
@@ -1,13 +1,26 @@
...
 
 public class Main {
 
+    //for check
     public static void main(String[] args) {
-        Calendar calendar = Calendar.getInstance();
-        System.out.println("Current Date and Time :");
-        System.out.format("%tB %te, %tY%n", calendar, calendar, calendar);
-        System.out.format("%tl:%tM %tp%n", calendar, calendar, calendar);
+        LocalDateTime date = LocalDateTime.now();
```
Comment (AnnaDremluga) [inline src/com/company/Main.java:17]: You should use variable as much close to the place, where it was created, as you can.

**Review by AnnaDremluga — CHANGES_REQUESTED** [review]

**Review by AnnaDremluga — APPROVED** [review]

**Pull request merged**